openapi/paths/usages.yaml

```
@@ -0,0 +1,68 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Usage
+  summary: Retrieve usage records
+  operationId: GetUsagesCollection
```

```suggestion
  operationId: GetUsageCollection
```

We name it like `GetCustomerCollection` (not `GetCustomersCollection`).